cmd/kubeops/internal/handler/dashboardcompat.go

```diff
@@ -0,0 +1,121 @@
+// The Dashboard doesn't understand the Helm 3 release format.
+// This file is a compatibility layer that translates Helm 3 releases to a Helm 2-similar format suitable for the Dashboard.
+
+package handler
+
+import (
+	"strings"
+
+	"github.com/golang/protobuf/ptypes/any"
+	"gopkg.in/yaml.v2"
+	h3chart "helm.sh/helm/v3/pkg/chart"
+	h3 "helm.sh/helm/v3/pkg/release"
+	h2chart "k8s.io/helm/pkg/proto/hapi/chart"
+	h2 "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+// generatedYamlHeader is prepended to YAML generated from the internal map[string]interface{} representation.
+const generatedYamlHeader = "# Not original YAML! Generated from parsed representation."
```

Comment:
I would skip this header, then in the dashboard we can try to recover the comments

Reply:
I think the header should be there for clarity until we can do something better, such as storing the YAML separately, as you suggested.

Reply:
Yes, the problem is that it's giving to the user internal information about a problem they may not be aware of. It seems like an error. It's easier to understand that the parsed YAML is what it's stored in the cluster (with no comments).

Reply:
I see!